resolver/resolver_test.go

```diff
@@ -2103,9 +2103,6 @@ func TestLiteral(t *testing.T) {
 					Build(t),
 			).SetAlias(ref.Field(t, "content", "Content", "message_field")).Build(t),
 		},
-	)
-	content.Fields = append(
```

Comment:
The diff looks a little confusing, but it just merges the two `append` into one.